llama-index-core/llama_index/core/storage/chat_store/base.py

```diff
@@ -46,3 +46,31 @@ def delete_last_message(self, key: str) -> Optional[ChatMessage]:
     def get_keys(self) -> List[str]:
         """Get all keys."""
         ...
+
+    async def aset_messages(self, key: str, messages: List[ChatMessage]) -> None:
+        """Async version of Get messages for a key."""
+        self.set_messages(key, messages)
+
+    async def aget_messages(self, key: str) -> List[ChatMessage]:
+        """Async version of Get messages for a key."""
+        return self.get_messages(key)
+
+    async def aadd_message(self, key: str, message: ChatMessage) -> None:
```

Comment:
small nit -- I would prefer `async def async_add_message(...)` -- the double `aa` makes it seem like a typo, and the vector store followed a similar pattern for naming the add function

Reply:
Sounds good